foam_internalwrapper/foam_dicts.py

```diff
@@ -6,6 +6,46 @@
 
 import simphonyfoaminterface as foamface
 from simphony.core.cuba import CUBA
+# from cuba_extension import CUBAExt
+
+dataNameMap = {CUBA.PRESSURE: "p",
```

Comment:
I like how the two `foam_mesh.py` files have been refactored so that you use this map instead of the many if-statements. :+1: 

My only suggestion would be to move these first ~50 lines from this file (`foam_internalwrapper/foam_dicts.py`) and from `foam_controlwrapper/foam_templates.py` to a single file as they are identical.

Or alternatively just add an issue to remind us that this should be done sometime soon (i.e. after this PR).